### PR TITLE
Magazine homework

### PR DESCRIPTION
**Pull request by daguirreglobant**
Sobreescribo los archivos de mi rama en la tarea 1 con la primera tarea de magazine.

### CODE REVIEW

**Comment by emolinaglobant** [comment]
@daguirreglobant , the title looks misaligned, the font size for the content does not look like the original cover, and looks like the height for the barcode is forced making it look weird

Screen Shot 2022-01-14 at 11 31 34 AM
.

File: daniel-aguirre-salamanca/tarea-1/styles.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    text-align: center;
+    color: #37beba;
+}
+
+.magazine__header span:last-child{
+    padding-right: 4rem;
+    text-align: right;
+}
+
+.magazine__content{
```
Comment (emolinaglobant) [inline daniel-aguirre-salamanca/tarea-1/styles.css:51]: @daguirreglobant remember to keep alphabetical order in your CSS rules

**Comment by emolinaglobant** [comment]
@daguirreglobant congrats on using BEM

**Comment by daguirreglobant** [comment]
@emolinaglobant I reorganized the css alphabetically and changed the background to the same as the original magazine. Added a result of how it looks in my screen as a "result_in_image.png". 

Here you can see a preview: https://raw.githack.com/emolinaglobant/academy-react/daniel-aguirre-salamanca/daniel-aguirre-salamanca/tarea-1/magazine.html

**Comment by emolinaglobant** [comment]
@daguirreglobant , nice work!

**Pull request merged**